python/adelphi/tests/unit/schema_util.py

```diff
@@ -0,0 +1,106 @@
+from cassandra.metadata import Metadata,\
+	KeyspaceMetadata,\
+	TableMetadata,\
+	ColumnMetadata,\
+	IndexMetadata,\
+	SimpleStrategy, \
+	UserType
+
```

Comment:
utility script that programmatically generates a C* schema compatible with C* from 2.1.22 to 4.0-beta3

it contains a couple of keyspaces, tables, indexes (regular and custom) and complex UDTs